portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/usecase/WYSIWYGUsecase.testcase

```diff
@@ -224,7 +224,8 @@ definition {
 			RobotType.robotTypeShortcut(locator1 = "Ctrl + v");
 		}
 
-		Pause(locator1 = "5000");
+		WaitForElementPresent(
+			locator1 = "WCEditWebContent#TEXT_HEADER");
```

Comment:
1. I don't like creating new product team paths where possible. It just adds to the fluff.
2. This doesn't properly wait for the specific pasted content

Create a new path in CKEditor.path instead of WC

```
BODY_TEXT
//body[contains(@class,'cke_editable') and contains(.,'${key_text}')]
```
and use
```
WaitForElementPresent(locator1 = "CKEditor#BODY_TEXT");
```